banks/serializers.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from .models import Bank, Branch, Account, Transaction, TransactionType
+
+
+class BankSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Bank
+        fields = '__all__'
+
+
+class AccountSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Account
+        fields = '__all__'
+
+
+class BranchSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Branch
+        fields = '__all__'
+
+
+class TransactionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Transaction
+        fields = '__all__'
+
+    def validate_amount(self, value):
```

Comment:
look into [this](https://stackoverflow.com/questions/39598576/how-to-validate-a-field-on-update-in-drf)